### ISSUE TITLE
Problems with essentia generators

### ISSUE THREAD

**Issue by drayath** [GTNewHorizons/GT-New-Horizons-Modpack#4945]
#### Which modpack version are you using?
2.0.7.5

#
#### If in multiplayer; On which server does this happen? 
Private

#
#### What did you try to do, and what did you expect to happen?
I have a essentia furnace producing ~1 potentia essentia per second

I have some Potentia Generators that use 1 essentia every 20 seconds

Setup with the  a line of generators with a essentia pipe running along side them (connected to each)

Expected that once the first generator completely filled up, the next would start filling and run, etc.

#
#### What happened instead? (Attach screenshots if needed)

The first generator consumes all the essentia rather than just the one every 20 seconds it needs even when internal store is full.
If you disconnect the pipe it correctly uses up one essentia every 20 seconds from its internal store, reconnect the pipe and the generator will consume all (1 per sec) essentia from the pipe.

In addition (and possibly by design). once the internal battery of the generator is full it still consumes essentia.

#
#### What do you suggest instead/what changes do you propose?

generators should only request/accept a unit of essentia when it requires a complete unit, not when internal tank it is down to 63.99 of 64

**Comment by drayath** [comment]
Just spent an afternoon and much of my spare resources building a stack of potentia generators, and now can't really use them  <sadness>

**Comment by bartimaeusnek** [comment]
well im gonna take a look at it and will fix it sooner or later, but currently im in work for some material generation, so even a quickfix for this will take some time.

**Comment by drayath** [comment]
No rush needed on my behalf, i'm on single player so i can just set up my generator room, pretend it works and cheat in a few EV geothermals and an infinite lava source to actually provide the correct power output.

**Comment by psuemno** [comment]
As somebody who loves magic, I did some testing on single player to try and understand how these things work. I was similarly frustrated by essentia tubes not behaving and only being able to fill up 1 generator at a time.

As a stop-gap measure I tried putting the essentia in jars just to store it for a bit, and lo and behold the essentia started flying through the air and all the generators lit up.  

I believe that these generators can pull from jars in the same way that Infusion Altars can, with a range of ~6 blocks. Instead of trying to feed them directly with tubes, just put your essentia in a nearby jar and see if that fixes things.
